features/config-is-true.feature

```diff
@@ -0,0 +1,113 @@
+Feature: Determine whether the value of a constant or variable defined in wp-config.php and wp-custom-config.php files is true.
+
+  Scenario: Get the value of a variable whose value is true
+    Given a WP install
+
+    When I run `wp config set WP_TRUTH true`
+    Then STDOUT should contain:
+    """
+    Success:
+    """
```

Comment:
```suggestion
    Then the return code should be 0
```
🔢 Optional: if we're just asserting our test setup is good, this would be fewer lines to the document overall. 

Damn i wish we could to table driven tests here 😆